CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Delta Chat Android Changelog
 
+## Unreleased
+
+* 'Show all email' is the default for chatmail, no need for an option
```

Comment:
Maybe frase it as: `hide superfluous "Show Classic E-mails" advanced setting for chatmail`

Reply:
i changed that line, thanks for the suggestion